### PR TITLE
2.8.4.2 Changelog

### PR DESCRIPTION
**Pull request by acgoldsmith**
### Description

https://konghq.atlassian.net/browse/DOCU-3270

### Testing instructions

Netlify link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

File: app/gateway/changelog.md [hunk 1]
```diff
@@ -2206,6 +2206,22 @@ openid-connect
...
+* Fixed a bug where subrequests produced by the `error_page` directive could interfere with worker processes when *buffered proxying* is enabled.
+
+#### Kong Manager
+* Fixed an issue where the Zipkin plugin didn’t allow the addition of `static_tags` through the Kong Manager UI. 
+* Fixed an issue where some of the icons were not rendering correctly.
+
+#### Plugins
+* Fixed an issue with the Oauth 2.0 Introspection plugin where a request with JSON that is not a table failed.
+* Fixed an issue where the slow startup of the Go plugin server caused a deadlock.
+* Fixed an internal error that occurred while validating the schema if `http_endpoint` contained userinfo but the headers were empty.
```
Comment (outsinre) [inline app/gateway/changelog.md:2223]: Probably, we should remove this line. I cannot locate the commit in `2.8.4.2-rc1`.

```suggestion
```

**Review by outsinre — APPROVED** [review]

**Review by lena-larionova — APPROVED** [review]

File: app/gateway/changelog.md [hunk 2]
```diff
@@ -2206,6 +2206,21 @@ openid-connect
 * Bumped `lodash` for Dev Portal from 4.17.11 to 4.17.21
 * Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21
 
+## 2.8.4.2
```
Comment (outsinre) [inline app/gateway/changelog.md:2209]: @acgoldsmith

Shall we add the two lines at https://github.com/Kong/kong-ee/blob/next/2.8.x.x/CHANGELOG-EE.md?plain=1#L11 to this PR?

**Review by acgoldsmith — COMMENTED** [review]
updating dependencies

File: app/gateway/changelog.md [hunk 3]
```diff
@@ -2228,6 +2228,23 @@ openid-connect
...
+
+**Release Date** 2023/06/30
```
Comment (acgoldsmith) [inline app/gateway/changelog.md:2233]:
```suggestion
**Release Date** 2023/07/07
```

**Review by acgoldsmith — COMMENTED** [review]
updating release date

**Comment by outsinre** [comment]
Hello @acgoldsmith , one of the changelog entry has been updated to give customers more accurate info. Please see https://github.com/Kong/kong-ee/pull/5944.

Could you kindly update accordingly?

File: app/gateway/changelog.md [hunk 4]
```diff
@@ -2228,6 +2228,23 @@ openid-connect
 * Bumped `lodash` for Dev Portal from 4.17.11 to 4.17.21
 * Bumped `lodash` for Kong Manager from 4.17.15 to 4.17.21
 
+## 2.8.4.2
+
+**Release Date** 2023/07/07
+
+### Fixes
+* Fixed a bug where subrequests produced by the `error_page` directive could interfere with worker processes when *buffered proxying* is enabled.
```
Comment (acgoldsmith) [inline app/gateway/changelog.md:2236]:
```suggestion
* Fixed a bug where internal redirects, such as those produced by the `error_page` directive,
  could interfere with worker process handling the request when *buffered proxying* is
  being used.
```

**Review by acgoldsmith — COMMENTED** [review]
syncing fix description

**Pull request merged**